### PR TITLE
Tno-1323

### PR DESCRIPTION
**Pull request by kylgarmor**
Initial setup for accessing a local copy/extract of the TNO database
Only the NEWS_ITEMS table has been implemented
Initial access to NEWS_ITEMS table via ContentMigration service

### CODE REVIEW

File: db/oracle/docker/scripts/04_data.sql
Comment (Fosol) [inline db/oracle/docker/scripts/04_data.sql:1]: @kylgarmor Can you confirm there is nothing private or secret in this file?

File: libs/net/services/Helpers/IngestManagerFactory`.cs [hunk 1]
```diff
@@ -35,7 +35,7 @@ public IngestManagerFactory(IServiceProvider serviceProvider)
     /// </summary>
     /// <param name="ingest"></param>
```
Comment (Fosol) [inline libs/net/services/Helpers/IngestManagerFactory`.cs:36]: Missing parameter in docs.  If you do a `dotnet build` it should show you a number of linting issues.

File: libs/net/services/Managers/IngestManager`.cs [hunk 2]
```diff
@@ -35,13 +37,15 @@ public abstract class IngestManager<TIngestServiceActionManager, TOption> : Serv
     /// <param name="options">Configuration options.</param>
     /// <param name="logger">Logging client.</param>
     public IngestManager(
+        IServiceProvider serviceProvider,
```
Comment (Fosol) [inline libs/net/services/Managers/IngestManager`.cs:40]: Missing parameter in docs

File: services/net/contentmigration/ContentMigrationService.cs [hunk 3]
```diff
@@ -34,17 +37,26 @@ public ContentMigrationService(string[] args) : base(args)
...
+
         // TODO: Figure out how to validate without resulting in aggregating the config values.
         // services.AddOptions<ContentMigrationOptions>()
         //     .Bind(this.Configuration.GetSection("Service"))
         //     .ValidateDataAnnotations();
 
+        // Build an intermediate service provider
+        var sp = services.BuildServiceProvider();
+        // This will succeed.
+        var msg = sp.GetService<MigrationSourceContext>();
```
Comment (Fosol) [inline services/net/contentmigration/ContentMigrationService.cs:58]: Don't need this anymore

**Review by Fosol — CHANGES_REQUESTED** [review]

File: db/oracle/docker/scripts/04_data.sql
Comment (kylgarmor) [reply db/oracle/docker/scripts/04_data.sql:1]: - the only names in here appear to be original article authors/pundits which are already public  
- urls refer to original article source on the web  
- both the `TEXT` and `BINARY` fields which are of type `CLOB` and `BLOB` are empty

**Review by Fosol — APPROVED** [review]

**Pull request merged**